python/sglang/srt/layers/moe/fused_moe_triton/fused_moe.py

```diff
@@ -578,6 +579,7 @@ def fused_experts_impl(
         else:
             raise ValueError(f"Unsupported activation: {activation=}, with {is_gated=}")
 
+        intermediate_cache3.fill_(0)
```

Comment:
It introduces a redundant kernel. Is it needed? We zero out padded tokens here: https://github.com/sgl-project/sglang/blob/227728e173baea3d2b7a0c394ac8883ccb2447ff/python/sglang/srt/layers/moe/fused_moe_triton/fused_moe_triton_kernels.py#L176-L192